ister_gui.py

```diff
@@ -3271,7 +3271,10 @@ def automatic_install(self):
         args = ister.handle_options(ister_cmd)
 
         ister.LOG = ister.logging.getLogger('ister')
-        ister.handle_logging(args.loglevel, ister_log, AlertLoggerHandler)
+        try:
+            ister.handle_logging(args.loglevel, ister_log, AlertLoggerHandler(title))
+        except Exception as e:
+            self.logger.debug(e)
```

Comment:
This seems like it wouldn't do much if the logger can't be setup. Also pretty unrelated to the change over all.

Reply:
I mean I don't know, I imagine that if somebody gave a logfile location they couldn't write to that it may be better to just fail rather than not log. I'm open to ideas.

Reply:
mmm ok, changing it